agent/bin/kontena-agent

```diff
@@ -31,6 +31,8 @@ Kontena::Logging.initialize_logger(STDOUT, log_level)
 
 agent = Kontena::Agent.new(
   api_uri: api_uri,
-  api_token: api_token
+  api_token: api_token,
+  ssl_verify: ENV['KONTENA_SSL_VERIFY'],
+  ssl_ca: ENV['KONTENA_SSL_CA'],
```

Comment:
NB: For the CLI `SSL_CERT_FILE=`, excon actually handles it: https://github.com/excon/excon/blob/v0.57.1/lib/excon/ssl_socket.rb#L34

I find the behavior of the libssl `SSL_CERT_FILE=` env in the `kontena/agent` container odd... it doesn't seem to be respected by the default cert store:

```
irb(main):015:0> ssl_cert = OpenSSL::X509::Certificate.new File.read(ENV['SSL_CERT_FILE'])
=> #<OpenSSL::X509::Certificate: subject=#<OpenSSL::X509::Name:0x0055eafa90aef8>, issuer=#<OpenSSL::X509::Name:0x0055eafa90af20>, serial=#<OpenSSL::BN:0x0055eafa90af48>, not_before=2017-07-06 08:07:56 UTC, not_after=2018-07-06 08:07:56 UTC>
irb(main):013:0> ssl_store = OpenSSL::X509::Store.new
=> #<OpenSSL::X509::Store:0x0055eafa9b6280 @verify_callback=nil, @error=nil, @error_string=nil, @chain=nil, @time=nil>
irb(main):014:0> ssl_store.set_default_paths
=> nil
irb(main):016:0> ssl_store.verify(ssl_cert)
=> false
```

Have to explicitly set the `ENV['SSL_CERT_FILE']` as a CA path for it to work:

```
irb(main):017:0> ssl_store = OpenSSL::X509::Store.new
=> #<OpenSSL::X509::Store:0x0055eafa8ecf70 @verify_callback=nil, @error=nil, @error_string=nil, @chain=nil, @time=nil>
irb(main):018:0> ssl_store.add_file ENV['SSL_CERT_FILE']
=> #<OpenSSL::X509::Store:0x0055eafa8ecf70 @verify_callback=nil, @error=nil, @error_string=nil, @chain=nil, @time=nil>
irb(main):019:0> ssl_store.verify(ssl_cert)
=> true
``` 

Same thing with `ssl_context.set_params` vs `ssl_context.set_params(ca_file: ENV['SSL_CERT_FILE'])`.